src/main/java/com/pickyfy/pickyfy/domain/BaseTimeEntity.java

```diff
@@ -0,0 +1,23 @@
+package com.pickyfy.pickyfy.domain;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.EntityListeners;
+import jakarta.persistence.MappedSuperclass;
+import java.time.LocalDateTime;
+import lombok.Getter;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+@MappedSuperclass
+@EntityListeners(AuditingEntityListener.class)
+@Getter
+public abstract class BaseTimeEntity {
+    @CreatedDate
```

Comment:
```suggestion
public abstract class BaseTimeEntity {

    @CreatedDate
```
클래스 선언과 필드 정의 사이에 빈 줄을 추가하는게 가독성이 더 좋을 거 같습니다~ 다른 클래스들도 변경 부탁드립니다